### ISSUE TITLE
Packit build failures after merge

### ISSUE THREAD

**Issue by praiskup** [fedora-copr/copr#2884]
We have "negative" CI karma for quite some time:
 https://github.com/fedora-copr/copr/commits/main

What if we disabled Packit for the time being for pushes?

cc @lachmanfrantisek @lbarcziova

**Comment by nikromen** [comment]
I can see failed builds since https://github.com/fedora-copr/copr/commit/fc73a835add608c13133b9024bbaf7a64715043d

perhaps it's due to the `~~dev` change we did in copr-common version

**Comment by nikromen** [comment]
So it seems like parsed spec file in https://download.copr.fedorainfracloud.org/results/packit/copr/fedora-39-x86_64/06266070-python-copr-common/ and its SRPM is different (look to the spec file in SRPM and spec file inside .tar.gz in SRPM - their version differs) - that's responsible for the majority of errors

another bug I found (some time ago) and is responsible for the first fail in https://copr.fedorainfracloud.org/coprs/packit/copr/package/python-copr-common/ is that versions in spec file in SRPM and in tar.gz (inside SRPM) sometimes differ - usually when I submit a PR with new pkg version then it has still its lower version and one more push to PR/new PR to catch up/refresh the version is needed

but I have no idea whether this is our or packit/specfile issue

**Comment by nikromen** [comment]
also, I don't think it's a ci-blocker since we have our CI still running in parallel

**Comment by praiskup** [comment]
> edit: I guess grep '"%version"' setup.py fails when build is submitted by packit

For some reason, Packit generates version `0.20.1`, while our specfile has `0.20`.

> also, I don't think it's a ci-blocker since we have our CI still running in parallel

Well I need to have green CI mark in recent commits, otherwise it blocks my brain. :-)

**Comment by praiskup** [comment]
#2956 likely fixes this!

**Comment by praiskup** [comment]
And all green, finally!

**Comment by praiskup** [comment]
Thanks @lachmanfrantisek :)
